### PR TITLE
Improve performance of dependency injection system by removing yaml parsing

### LINKED ISSUE #16005: More performance, speed first
```
xdebug.profiler_enable=0
xdebug.profiler_enable_trigger=1
xdebug.profiler_append=1
xdebug.profiler_output_dir="/mnt/Dev/@phpmyadmin/theREALphpMyAdminREPO"
xdebug.profiler_output_nme="xdebug-profile.%t.%p"

xdebug.auto_trace=1
xdebug.trace_enable_trigger=1
xdebug.trace_options=1
xdebug.collect_params=4
xdebug.collect_return=0
xdebug.trace_format=0
xdebug.show_mem_delta=1
xdebug.trace_output_dir="/mnt/Dev/@phpmyadmin/theREALphpMyAdminREPO"
xdebug.trace_output_name="trace.%t.%p"

xdebug.file_link_format="xdebug://%f@%l"
xdebug.remote_log="/mnt/Dev/@phpmyadmin/theREALphpMyAdminREPO/xdebug.log"
```

Yes I know we are talking about milliseconds. This is the point of this issue.

Symfony DependencyInjection consumes to much time.

### PR DESCRIPTION
**Pull request by williamdes**
### Description

Ref #16005

### CODE REVIEW

File: libraries/services.php [hunk 1]
```diff
@@ -0,0 +1,277 @@
+<?php
+
+return [
+    'services' =>
+    [
+        'advisor' =>
+        [
+            'class' => 'PhpMyAdmin\Advisor',
```
Comment (williamdes) [inline libraries/services.php]: I think `::class` could be used
Do I make the change @mauriciofauth ?

File: libraries/services_loader.php [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+namespace Symfony\Component\DependencyInjection\Loader\Configurator;
+
+use Symfony\Component\DependencyInjection\Reference;
+
+return function(ContainerConfigurator $configurator) {
+    $services = $configurator->services();
+
+    $servicesFile = include __DIR__ . '/services.php';
+    foreach ($servicesFile['services'] as $serviceName => $service) {
```
Comment (williamdes) [inline libraries/services_loader.php]: I will de-dupe the code if the PR is approved

File: libraries/services.php (hunk 1 above)
Comment (mauriciofauth) [reply libraries/services.php]: I agree.

File: libraries/services_loader.php (hunk 2 above)
Comment (mauriciofauth) [reply libraries/services_loader.php]: The nesting can be reduced by using early exits.

**Review by MauricioFauth — APPROVED** [review]

File: libraries/services.php (hunk 1 above)
Comment (williamdes) [reply libraries/services.php]: Done

File: libraries/services_loader.php (hunk 2 above)
Comment (williamdes) [reply libraries/services_loader.php]: Done in 07e4fbc0559e238cca02731485fbe599094ef9c3

**Review by ibennetch — APPROVED** [review]
👍

**Comment by williamdes** [comment]
Thank you for the review, I really appreciate ;)

**Pull request merged**